nectar/src/config/file.rs

```diff
@@ -57,7 +57,7 @@ pub struct BitcoinFee {
     /// The static value to use if the selected strategy is "static"
     #[serde(default)]
     #[serde(with = "crate::config::serde::bitcoin_amount::sat_as_optional_unsigned_int")]
```

Comment:
Now that we are just re-exporting `rust-bitcoin`'s `Amount`, I think we should be able to use this module here: https://github.com/rust-bitcoin/rust-bitcoin/blob/c1ae3b795547176d1403e12a6f708ffecfa52c1d/src/util/amount.rs#L938-L960

Reply:
> Now that we are just re-exporting `rust-bitcoin`'s `Amount`, I think we should be able to use this module here: [rust-bitcoin/rust-bitcoin@`c1ae3b7`/src/util/amount.rs#L938-L960](https://github.com/rust-bitcoin/rust-bitcoin/blob/c1ae3b795547176d1403e12a6f708ffecfa52c1d/src/util/amount.rs#L938-L960)

Oh, thanks, I did not know this one! I assumed they did not have an opt one.